File: lista2sala/3.py
a=float(input("digite o primeiro valor numérico:"))
b=float(input("digite o segundo valor numérico:"))
if a==b:
    print("os números digitados são iguais, então não há diferenças entre eles")
else:
    if a<b:
        dif=b-a
    else:
        dif=a-b
    print("a diferença entre os números é:", dif)

File: lista2sala/3.c
#include <stdio.h>
int main(){
    int a, b, d;
    printf("digite os valores numericos:");
    scanf("%d%d", &a, &b);
    if (a>b){
        d=a-b;
        printf("a diferença é %d", d);
    }
    else{
        d=b-a;
        printf("a diferença e: %d", d);
    }
    
}